packages/components/status/stories/status.happo.js

```diff
@@ -0,0 +1,19 @@
+import { storiesOf } from '@open-wc/demoing-storybook';
+import '@tradeshift/elements';
+import '@tradeshift/elements.status';
+
+import { createHappoStories } from '../../../../.storybook-happo/utils';
+import { STATUS_TYPE } from '../src/utils/constants';
+
+storiesOf('ts-status', module).add('test', () => {
+	const properties = {
+		status: STATUS_TYPE,
+		text: 'test'
+	};
+
+	const options = {
+		columns: 5
```

Comment:
Can you try to change this value to 2 or even remove it.
This value affects the rendering of the happo visual tests and normally it solves the issue when we have too many properties, but here you have only two properties so it could be rendered in one column.

Reply:
ok